en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md

```diff
@@ -5,7 +5,7 @@ comprehensive human capital management system that covers the entire employee li
 benefits, talent management, workforce management, and services. The entire system resides on cloud that takes the 
 burden of managing and replicating data on-premise.
 
-## What you'll build
+## What You'll Build
```

Comment:
Use sentence style for titles.  https://learn.microsoft.com/en-us/style-guide/text-formatting/formatting-titles